README.md

```diff
@@ -82,7 +82,10 @@ Example:
 ```js
 'use strict';
 const Model = require('screwdriver-models');
-const factory = Model.PipelineFactory.getInstance({ datastore });
+const factory = Model.PipelineFactory.getInstance({
+    datastore,
+    scmPlugin,          // scm plugin.  For example, screwdriver-scm-github
```

Comment:
The example is a bit wonky. Simply stating "scm plugin" isn't very descriptive. Can we just require `screwdriver-scm-github` if that's going to be your example anyways?